netci.groovy

```diff
@@ -44,12 +44,19 @@ def newVsiJob = job(newVsiJobName) {
 
     // This opens the set of build steps that will be run.
     steps {
-        // Build roslyn-project-system repo.
-        batchFile("build.cmd /release")
+        // Build roslyn-project-system repo - we also need to set certain environment variables for building the repo with VS15 toolset.
+        batchFile("SET VS150COMNTOOLS=%ProgramFiles(x86)%\Microsoft Visual Studio\VS15Preview\Common7\Tools\
```

Comment:
These environment variables seemed to be not set for the snapped willow drop - I am not sure if this is fixed in future drops. We can remove this workaround once we move to a willow drop which has these set by default on install.